apps/hannk/interpreter/ops.cpp

```diff
@@ -1400,12 +1400,16 @@ void ReshapeOp::execute() {
         assert(new_shape.at(d) == output_buf.dim(d).extent());
     }
 
-    assert(input_buf.number_of_elements() == output_buf.number_of_elements());
+    // NOTE: HCHECK, not assert, to ensure a runtime failure to match TFLite
```

Comment:
We should still be asserting here. This check should be in the parser (i.e. not running every iteration).

Reply:
The dynamic-tensor case still needs a runtime check.